test/README

```diff
@@ -76,12 +76,13 @@ You can set these environment variables to configure the test suite:
   TEST_OS    The OS to run the tests in.  Currently supported values: (vm-create)
                 "fedora-23"
                 "fedora-24"
+                "fedora-25"
                 "fedora-testing"
                 "fedora-atomic"
                 "centos-7"
                 "rhel-7"
                 "debian-unstable"
-             "fedora-24" is the default (testinfra.py, testlib.sh)
+             "fedora-25" is the default (testinfra.py, testlib.sh)
```

Comment:
Also, testlib.sh is now testlib.py, but that now gets its defaults properly from testinfra